src/freerct.cpp

```diff
@@ -125,13 +125,9 @@ static void MigrateOldFiles()
 			{homedir + "" + DIR_SEP + ".config" + DIR_SEP + "freerct",
 			homedir + DIR_SEP + ".local" + DIR_SEP + "share" + DIR_SEP + "freerct",
 			homedir + DIR_SEP + ".freerct"}) {
-		std::unique_ptr<DirectoryReader> dr(MakeDirectoryReader());
-		dr->OpenPath(old_directory.c_str());
-		for (;;) {
-			const char *filename = dr->NextEntry();
-			if (filename == nullptr) break;
 
-			std::string name(filename);
+		for (std::string name : GetAllEntries(old_directory)) {
```

Comment:
```suggestion
		for (const std::string &name : GetAllEntries(old_directory)) {
```

Reply:
Dropped const here because the value is modified afterwards. Everything else should be fixed now, including tidy problem :)